File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.java
class Solution {
    public int firstUniqChar(String s) {
        int i=0;
        for (i=0; i<s.length(); i++) {
            int j=0;
            for (j=0; j<s.length(); j++) {
                if (s.charAt(i)==s.charAt(j) && i!=j) break;
            }
            if (j==s.length())
            return i;
        }
        return -1;
    }
}

File: 0387-first-unique-character-in-a-string/0387-first-unique-character-in-a-string.py
class Solution:
    def firstUniqChar(self, s: str) -> int:
        freq = {}
        for i in range(len(s)):
            if s[i] not in freq:
                freq[s[i]] = i
            else:
                freq[s[i]] = -1
        for v in freq.values():
            if v != -1: return v
        return -1